app/assets/stylesheets/leaderboard.css

```diff
@@ -221,6 +248,13 @@
   font-size: calc(16 * var(--fpx));
 }
 
+.leaderboard-item.bar-data>*{
+  position: relative;
+  max-width: 100%;
+  overflow: hidden;
+  white-space: nowrap;
+}
+
```

Comment:
Not sure where to leave this comment - 
There's currently no padding under the 'g's in the carbon emissions columns when hovering over each entry. If we're keeping the units here (i.e. not moving them to the header) could we increase the top and bottom padding maybe?

![image](https://github.com/openflighthpc/carbon-leaderboard/assets/102584263/6e29452a-407a-48e1-a56f-0e8aba61d421)

Reply:
Can I round this by using a smaller font size for the units? Cuz changing the height of the wrapper will affect the layout of other elements in the wrapper

Reply:
Sure that sounds good :)